### DISCUSSION TITLE
Session recording keep duration

### DISCUSSION THREAD

**Discussion by maticboncina** [gravitational/teleport#7475] [Q&A] [answered]
Hello,

Currently, my session recordings get deleted after 24 hours. How can I extend that to something like 90 days?

**Comment by webvictim** [comment, marked as answer]
Session recordings are actually kept forever, but the web UI only displays today's recordings by default. You can use the time selector in the top right to extend the range:

![Screenshot 2021-07-12 at 13 40 29](https://user-images.githubusercontent.com/897821/125325041-e2df4200-e316-11eb-84e0-0ef048e606ed.png)

![Screenshot 2021-07-12 at 13 40 35](https://user-images.githubusercontent.com/897821/125325023-dbb83400-e316-11eb-96c3-224e7e088c68.png)

**Reply by r-kjha** [reply]
How to delete a recording or make it so that the recordings are deleted after a certain number of days?

**Reply by webvictim** [reply]
You can delete recordings from the audit backend where Teleport keeps them - `/var/lib/teleport/log/records` for on-disk storage, or DynamoDB/Firestore if using either of those.

There is currently no built-in way to automatically delete session recordings after a period of time.

**Comment by r-kjha** [comment]
How to delete a server from my teleport dashboard?

**Reply by webvictim** [reply]
Stop the Teleport agent on the machine and it will be removed from the dashboard after 10-15 minutes.
